openedx/features/course_experience/static/course_experience/js/CourseOutline.js

```diff
@@ -82,12 +82,12 @@ export class CourseOutline {  // eslint-disable-line import/prefer-default-expor
         toggleAllButton.setAttribute('aria-expanded', 'false');
         sectionAction = collapseSection;
         toggleAllSpan.classList.add(extraPaddingClass);
-        toggleAllSpan.innerText = 'Expand All';
+        toggleAllSpan.innerText = gettext('Expand All');
       } else {
         toggleAllButton.setAttribute('aria-expanded', 'true');
         sectionAction = expandSection;
         toggleAllSpan.classList.remove(extraPaddingClass);
-        toggleAllSpan.innerText = 'Collapse All';
+        toggleAllSpan.innerText = gettext('Collapse All');
```

Comment:
What about `Collapse All`? Does it have the translation?

Reply:
Yes, it has https://github.com/0lha/edx-platform/blob/ginkgo-rg/conf/locale/ru/LC_MESSAGES/djangojs.po#L7276

There was ru translations for "Expand All"/"Collapse All" in djangojs.po, but translation for django.po was missed, so button haven't been translated until click.
I have another thought. Should I check the same situation for other languages and transfer data from djangojs.po to django.po if necessary?

Reply:
got it, thanks